### ISSUE TITLE
Fork CMS 5.9.2 Composer issue.

### ISSUE THREAD

**Issue by GaneshKandu** [forkcms/forkcms#3250]
## Bug Reportvendor/autoload.php is missing from vendor folder.  

| Q            | A     |
| ------------ | ----- |
| Critical Bug | yes|
| Version      | 5.9.2 |

### Problem description

```
Warning: require(vendor/autoload.php): failed to open stream: No such file or directory in autoload.php on line 9 Warning: require(vendor/autoload.php): failed to open stream: No such file or directory in autoload.php on line 9 Fatal error: require(): Failed opening required 'vendor/autoload.php' in autoload.php on line 9 
```

### Steps to reproduce
Try to install Fork CMS

### Proposed solution

composer is not installed correctly. should installed correct.

**Comment by carakas** [comment]
@GaneshKandu 
is this a download from the website? or did you install it using git?
Can you fill in the version this applies to in the placeholder in the issue?

**Comment by GaneshKandu** [comment]
> is this a download from the website? or did you install it using git?
I downloaded from website
https://www.fork-cms.com/download

**Comment by Sellato** [comment]
the same :(

missing directories:

### docs
### vendors

in 

https://github.com/forkcms/forkcms/archive/5.9.2.zip
https://github.com/forkcms/forkcms/archive/5.9.2.tar.gz
 

![Screen-2020-12-10_15-36-26](https://user-images.githubusercontent.com/5456419/101745806-ddf6ad80-3afd-11eb-839f-b60e3c29de3f.jpg)

**Comment by carakas** [comment]
on the archives in Github it is normal, you just need to run composer install

**Comment by carakas** [comment]
I'll make a new release asap, made some changes on the build server. Should be working this time

**Comment by Sellato** [comment]
Some new problems :(

Incorrect directory Vendor

 missing `autoload.php`

**Comment by carakas** [comment]
Fixed it for the 5.9.2 download, should be fixed in future releases as well

**Comment by carakas** [comment]
@Sellato the source zips from github never have an installed composer included
